### ISSUE TITLE
h5p-editor.ts:411 Uncaught (in promise) ReferenceError: H5PEditor is not defined

### ISSUE THREAD

**Issue by stanlee321** [Lumieducation/H5P-Nodejs-library#1085]
Hi.

I was trying to implement the react + express-server combination, I successfully generate the `loadContentCallBack(contentId: string): Promise<IEditorModel>` from the server, but when the react frontend receives this information, shows the error in the title.

I attach a picture of the error with some aditional information.

[https://www.dropbox.com/s/22mqufwv91f7lzw/Screenshot%20from%202021-02-22%2019-22-46.png?dl=0](url)

Any glues about why this error happen??, I follow the base code in the react example.

Thank you!!.

**Comment by sr258** [comment]
Hi @stanlee321, 

it look like the express server doesn't serve the correct H5P core files. Make sure that you serve the H5P core files at `/h5p/core` and `/h5p/editor`. (https://docs.lumi.education/usage/integrating#serving-static-h-5-p-core-files-for-the-client) 

If you click on one of the core files in the network dev tool you can also check what response you got from the server. Is it possible that you serve a HTML document here (the console errors seem to indicate this).

**Comment by stanlee321** [comment]
Hi, 

The H5P core files seems to be working fine, all are reached with 200 status code, for be sure I switch to the example running  the demo for express server in `packages/h5p-examples/src/express.ts` and implement  the `loadContentCallBack(contentId: string): Promise<IEditorModel>` as is expected 

```javscript 
    // Content Model
    server.get('/contentModel', cors(options), async (req, res) => {

    
        h5pEditor.setRenderer((mod) => mod)

        const model = await h5pEditor.render(
            undefined,
        );

        //console.log(model)
        // console.log(model)
        res.send(model);
        res.status(200).end();
    });
```
But I get the same result. It seems to be a problem with this error   `Uncaught SyntaxError: Unexpected token '<'` where React Typescript cannot read the .js files  while is running in dev.

**Comment by sr258** [comment]
The /contentModel route is fine to start the editor when creating new content. Please check what the content of the downloaded .js files is when the browser tries to open them. They should not result in the error if they were downloaded correctly.

Have you also checked out the rest sample at  https://github.com/Lumieducation/H5P-Nodejs-library/tree/webcomponents/xapi-callback2/packages?

Can you share your repo, so that I can see a bit more of what‘s going on.

**Comment by stanlee321** [comment]
Hello.

Wow, I did not know that that branch existed, thank you very much, I was able to make it work using the example code of that branch, also I change the components to functional components and hooks and it works wonderfully. Again, thank you very much, although I could not find the solution to run the problem with my first implementation using `H5PEditorUI.tsx` as indicated in the h5p-react documentation. Anyway I leave here that component that I try to use within the repository in which I have everything working. Thank you.
`https://github.com/stanlee321/h5p-react-demo/blob/master/src/components/h5p/index.tsx`
